test-support/helpers/201-created/raw/expect-component.js

```diff
@@ -36,12 +39,12 @@ export default function(appOrContainer, expectation, count, options, customMessa
     }
   });
 
-  var message = count ? 
-                'Expected to find '+count+' components of type ' + expectation + '. Found: ' + found : 
+  var message = count ?
+                'Expected to find '+count+' components of type ' + expectation + '. Found: ' + found :
                 'Expected to find at least one component: ' + expectation;
```

Comment:
Oh I see what you mean, this PR removes the extra whitespace. Great, thanks for that.